app/controllers/sessions_controller.rb

```diff
@@ -1,20 +1,43 @@
 class SessionsController < ApplicationController
-    def new 
-    end 
+  def new
 
-    def create 
```

Comment:
Had to modify this, as I do not think we were actually finding the user before

Reply:
Good catch!